loader/CMakeLists.txt

```diff
@@ -169,6 +166,7 @@ if(WIN32)
         add_library(loader-unknown-chain OBJECT unknown_ext_chain_masm.asm)
         target_link_libraries(loader-unknown-chain Vulkan::Headers)
         target_include_directories(loader-unknown-chain PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
+        set_target_properties(loader-unknown-chain PROPERTIES LINKER_LANGUAGE C)
```

Comment:
Why is this needed now? I've seen this property set multiple times incorrectly in our CMake codebases. If it's actually needed please comment.

Reply:
This was because cmake could error out with
```cmake
CMake Error: CMake can not determine linker language for target: loader-unknown-chain
```
in some rare cases/configuration. Although I think it may be possible to get around that, but it may take more changes with `loader-opt`

Reply:
I think I will remove this change from this PR for now until I can narrow down potential options